### ISSUE TITLE
Alpha fade combat status

### ISSUE THREAD

**Issue by k1ck3r** [Sticklord/oUF_Abu#38]
hey,

could you be so kind and advice me where (or could it be done) to configure the alpha fade option for player/target? Asking this, as i use your unitframes on almost mid screen with some auras, and need to adjsut this. 

any help will be much appreciate
thanks in advance

**Comment by Sticklord** [comment]
Yeah it's a good idea, not quite sure how to handle it though. When should the frames be hidden?

Out of combat, have a target, not at max health?

**Comment by k1ck3r** [comment]
out of combat, no target, full hp is a good start
